<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomePage</class>
 <widget class="QWidget" name="HomePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="HomePageScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>2</y>
     <width>822</width>
     <height>528</height>
    </rect>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="widgetResizable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>820</width>
      <height>528</height>
     </rect>
    </property>
    <widget class="QPushButton" name="forwardUpBtn">
     <property name="geometry">
      <rect>
       <x>384</x>
       <y>10</y>
       <width>54</width>
       <height>38</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>54</width>
       <height>38</height>
      </size>
     </property>
    </widget>
    <widget class="QComboBox" name="currencyTypeBox">
     <property name="geometry">
      <rect>
       <x>103</x>
       <y>25</y>
       <width>112</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentText">
      <string/>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
     </property>
     <property name="minimumContentsLength">
      <number>0</number>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="modelColumn">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>28</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Currency:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="forwardDownBtn">
     <property name="geometry">
      <rect>
       <x>384</x>
       <y>480</y>
       <width>54</width>
       <height>38</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>54</width>
       <height>38</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HomePageScrollArea</class>
   <extends>QScrollArea</extends>
   <header>homepagescrollarea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
